### PR TITLE
Update for preferred Delta V exterior lights

### PR DESCRIPTION
**Pull request by IamVelcroboy**
## About the PR
See title.

## Why / Balance
The WD exterior lights are way too bright and wash out too much color compared to the Nyano ones. Rather than edit every file that has them or that comes in from upstream, this just migrates them out. Easy peasy.

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Lighting/base_lighting.yml [hunk 1]
```diff
@@ -149,7 +149,7 @@
 - type: entity
   id: PoweredlightExterior
   description: "A light fixture. Draws power and produces light when equipped with a light tube."
-  suffix: Blue
+  suffix: Blue, Don't Map on Delta V #Delta V: Don't Map on Delta V
```
Comment (DebugOk) [inline Resources/Prototypes/Entities/Structures/Lighting/base_lighting.yml]:
```suggestion
  suffix: Blue
  noSpawn: true # DeltaV - Don't map these
```

Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Structures/Lighting/base_lighting.yml]: oh, good call. Forgot all about that.

Comment (DebugOk) [reply Resources/Prototypes/Entities/Structures/Lighting/base_lighting.yml]: If you're automatically migrating them anyways why bother even allowing them to be mapped at all

**Comment by IamVelcroboy** [comment]
Wait, the tubes got lost somehow...oh nevermind. I see what it...going to add nospawn there as well.

File: Resources/Prototypes/Entities/Objects/Power/lights.yml [hunk 2]
```diff
@@ -190,7 +190,8 @@
   parent: BaseLightTube
   name: exterior light tube
   description: A high power high energy bulb for the depths of space. May contain mercury.
-  id: ExteriorLightTube
+  id: ExteriorLightTubeBlue
```
Comment (DebugOk) [inline Resources/Prototypes/Entities/Objects/Power/lights.yml]: Why is the id changing?

Comment (IamVelcroboy) [reply Resources/Prototypes/Entities/Objects/Power/lights.yml]: I'm already pushing. No idea why it did that

**Review by DebugOk — DISMISSED** [review]

**Pull request merged**